### ISSUE TITLE
More SORT options

### ISSUE THREAD

**Issue by TheMysticS** [d4rken-org/permission-pilot#123]
Can you add the following SORT options in the Apps Tab:

1. Size
2. RAM
3. Battery
4. Data

Thanks.

**Comment by d4rken** [comment]
Feels out of scope, like #122

**Comment by TheMysticS** [comment]
> Feels out of scope, like #122

Except RAM, the others should be straightforward, isn't it?

There are not too many sort options currently, if space is a constraint.

**Comment by dragonGod97677** [comment]
Kinda out of scope for a Permission manager app

**Comment by TheMysticS** [comment]
> Kinda out of scope for a Permission manager app
>
There is plenty space in the sort section. Do you have other sort categories that will be more useful?

**Comment by d4rken** [comment]
> > Kinda out of scope for a Permission manager app
> 
> There is plenty space in the sort section. Do you have other sort categories that will be more useful?

We do not have to fill every space.

**Comment by dragonGod97677** [comment]
> > Kinda out of scope for a Permission manager app
> 
> There is plenty space in the sort section. Do you have other sort categories that will be more useful?

Yes there are space or not but putting it on a app like this, might just redo this entirely to a ultimate app manager.
I personally, dont like what other developers, I'll mention one, like the developer of SHAREit, what they did to the app, bloated it with unnecessary things like adding a free news feed like but videos, sports news and other more probably unrelated to file sharing app

**Comment by TheMysticS** [comment]
You're not adding anything here. Only adding more viewing options. Filters and Sort options don't add bloat to an app. They only make it easier for the user to see info more clearly.

**Comment by dragonGod97677** [comment]
> You're not adding anything here. Only adding more viewing options. Filters and Sort options don't add bloat to an app. They only make it easier for the user to see info more clearly.

Some of my words are should be in #122 but I've posted here..

As for sorting and filtering like these 

App name
App Package name
Type of permission
Is user app
Is system app
Installed by *insert app store here*

And those are already in the app is should enough

**Comment by TheMysticS** [comment]
Talking of Ultimate App Manager, check this out:

GitHub - MuntashirAkon/AppManager: A full-featured package manager and viewer for Android
https://github.com/MuntashirAkon/AppManager


This is a very comprehensive app, and extremely well laid out. But unfortunately, it is overwhelming for the average user. 

And what I (despite considering myself as an above-average user) also didn't like is how it shows up pretty much every context menu, changes all the default apps, etc.

But for an advanced user, this is a superb app.
